tasks/build.yml

```diff
@@ -0,0 +1,48 @@
+---
+# This is included in deploy.yml only if {{ gmrender_resurrect_binary_url }} is not defined
+
+- name: Install build dependenices
+  apt:
+    name:
+      - autoconf
+      - automake
+      - libtool
+      - pkg-config
+      - libupnp-dev
+      - libgstreamer1.0-dev
+
+- name: Download gmrender-resurrect archive
+  get_url:
+    url: "{{ gmrender_resurrect_archive_url | default('https://github.com/' + gmrender_resurrect_github_repo + '/archive/' + gmrender_resurrect_version + '.tar.gz') }}"
+    dest: "{{ global_cache_dir }}/gmrender-resurrect-{{ gmrender_resurrect_version }}.tar.gz"
+    checksum: "{{ gmrender_resurrect_checksum }}"
+  become: false
+  delegate_to: localhost
+
+- name: Clear unarchive target directory
+  file:
+    path: /tmp/gmrender-resurrect-{{ gmrender_resurrect_version }}
+    state: absent
+
+- name: Unpack gmrender-resurrect archive
+  unarchive:
+    src: "{{ global_cache_dir }}/gmrender-resurrect-{{ gmrender_resurrect_version }}.tar.gz"
+    dest: /tmp
+
+- name: Compile gmrender-resurrect
+  shell: ./autogen.sh && ./configure && make
+  args:
+    chdir: /tmp/gmrender-resurrect-{{ gmrender_resurrect_version }}
+
+- name: Create deployment target directory
```

Comment:
This is already done in deploy.yml

Reply:
But for some and I don't feel like complicating this role.

Reply:
`build.yml` shouldn't depend on what some arbitrary other module does.